### PR TITLE
100% coverage

### PR DESCRIPTION
**Pull request by stripedpajamas**

### CODE REVIEW

File: lib/ilp.js [hunk 1]
```diff
@@ -100,12 +100,6 @@ class Ilp {
 
     this.log.info('Closing connection to ILP connector...')
     await connection.end()
-
-    // connection will probably disconnect the plugin automatically
-    if (plugin.isConnected()) {
```
Comment (joelwass) [inline lib/ilp.js:105]: why did you remove this?

**Review by joelwass — APPROVED** [review]

File: lib/ilp.js (hunk 1 above)
Comment (stripedpajamas) [reply lib/ilp.js:105]: Oh ya I meant to mention that in the PR desc. I confirmed the "probably" -- the connection auto closes the plugin on end.
